### PR TITLE
Move RefTypes and BulkMem to finished proposals list

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

**Review by rossberg — APPROVED** [review]
LGTM, though for process reasons, I suggest holding off landing this PR until the proposals have actually been merged. The WG chairs are currently preparing the official announcement which will start a 10 day period of objection after which that can happen.

File: finished-proposals.md
Comment (dulanov) [inline finished-proposals.md:22]: May be will be also good to expand previous WGs with year - 2020.

**Comment by protheory8** [comment]
Is this safe to merge now?

**Comment by dschuff** [comment]
Yes, this is good to go.

**Review by dschuff — APPROVED** [review]

**Pull request merged**
